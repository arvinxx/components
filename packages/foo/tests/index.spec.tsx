import React from 'react';
import renderer from 'react-test-renderer';

import Foo from '@arvinxu/modules-foo';

test('Foo', () => {
  const component = renderer.create(<Foo />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
