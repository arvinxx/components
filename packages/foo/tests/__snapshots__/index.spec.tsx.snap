// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foo 1`] = `
<div>
  Hello! 
</div>
`;
